### ISSUE TITLE
Resize

### ISSUE THREAD

**Issue by GeorgeRogner** [sqlitebrowser/sqlitebrowser#2753]
The developer could improve resize from Browse data to Windows 10 32 Bit OS. Almost it don't give to see the last records. It's a great application.

Useful extra information
-------------------------
> DB4S v3.12.2 [built for i386-little_endian-ilp32] on Windows 10 (10.0) (winnt/10.0.14393) [i386]
> using SQLite Version 3.35.5
> and Qt 5.12.8

**Comment by chrisjlocke** [comment]
Any chance of a screenshot to confirm the issue you're facing? Not sure I understand the resize issue and what you're seeing.

Images can be pasted directly in the comment or reply.

**Comment by GeorgeRogner** [comment]
![01](https://user-images.githubusercontent.com/86381877/123174063-ee48f700-d455-11eb-8bc5-48c3ef793ac9.jpg)
![02](https://user-images.githubusercontent.com/86381877/123174086-f4d76e80-d455-11eb-94f0-22318878793e.jpg)
![03](https://user-images.githubusercontent.com/86381877/123174096-f86af580-d455-11eb-8136-57efd2fa027c.jpg)

**Comment by GeorgeRogner** [comment]
1366 x 768 Resolution of the Video Monitor.
Thank you very so much, for his feedback.

**Comment by GeorgeRogner** [comment]
Version before.

In this version, I get to see the all records.

1366 x 768 Resolution of the Video Monitor.

![01](https://user-images.githubusercontent.com/86381877/123174922-4b917800-d457-11eb-9a82-4e8de9649a5b.jpg)
![02](https://user-images.githubusercontent.com/86381877/123174933-50562c00-d457-11eb-8002-fdf330678ba0.jpg)
![03](https://user-images.githubusercontent.com/86381877/123174935-53e9b300-d457-11eb-8016-62c16106deb2.jpg)

**Comment by GeorgeRogner** [comment]
It could to have the dark mode.

**Comment by justinclift** [comment]
Interesting.  1366 x 768 is still a very common laptop resolution, so we should probably make sure it works. :smile:

**Comment by chrisjlocke** [comment]
Interesting, as I don't get this on my laptop, which has similar (I believe) resolution.
What happens if you 'normalise' the window (click the middle button to restore it to normal size) then click it again to maximise it?

Is this your only screen, or do you have two screens?

> It could to have the dark mode

There is a dark mode, if you click 'Edit' and 'Preferences'. 

![image](https://user-images.githubusercontent.com/7872218/123217076-18380300-d4c2-11eb-8253-2d639d4e0eb7.png)

Change the 'Application style' dropdown to 'Dark mode'.

**Comment by chrisjlocke** [comment]
I 'sort of' replicated this if I set my Windows resolution to 800x600.  DB4S has a minimum size (you can't make the window smaller than 768 in height if 'normalised', but its happy to be smaller if maximised (bit weird...) but at 800x600, it still works.

Bits are off-screen at 800x600 'normalised'

![image](https://user-images.githubusercontent.com/7872218/123218298-73b6c080-d4c3-11eb-943f-2a37cec515f8.png)

But OK maximised.

![image](https://user-images.githubusercontent.com/7872218/123218485-af518a80-d4c3-11eb-9e16-02a6702c661b.png)

![image](https://user-images.githubusercontent.com/7872218/123218485-af518a80-d4c3-11eb-9e16-02a6702c661b.png)

While playing with resolutions, it was easy to 'break' it, so it didn't look right maximised (a bit like how you had it), but normalising it (making it a normal window) and maximising it returned it to normal.

Do you put your laptop to sleep by closing the lid?  I've found when this occurs, this can mess with the resolution slightly - the lock screen for example can be a small resolution, so when you log back in, windows 'jumps around' as it readjust the resolution.

**Comment by mgrojo** [comment]
> ![03](https://user-images.githubusercontent.com/86381877/123174096-f86af580-d455-11eb-8136-57efd2fa027c.jpg)

@GeorgeRogner, you have three rows of docks there, at the right. What happens if you activate "View > Window Layout > Reset Window Layout Alt+0"?

**Comment by chrisjlocke** [comment]
I'll close this due to lack of response.
@GeorgeRogner - Please comment if you're still having issues and this can be reopened.

**Comment by Walaryne** [comment]
Hello there, I'm having the exact same issue as OP.
Found this issue looking for a solution.
My laptop has 1366x768 resolution.
The layout works perfectly fine when "Remote" is disabled in the views. However once I enable it the issue happens again.

**Comment by chrisjlocke** [comment]
@Walaryne - What version of DB4S are you using?
I know it was a Windows thing, but have you tried maximising the window and restoring it to normal?

**Comment by mgrojo** [comment]
I think we already did something to improve this situation. Is this reproducible using the nightly builds or recent own compilation?

**Comment by Walaryne** [comment]
> @Walaryne - What version of DB4S are you using? I know it was a Windows thing, but have you tried maximising the window and restoring it to normal?

![image](https://user-images.githubusercontent.com/14822020/149188253-acd4d851-58a2-4731-98d9-fb0d701312c2.png)

Here's the version info that I'm using. Yes, I've tried resizing/maximizing minimizing a few times, and the issue persists.



> I think we already did something to improve this situation. Is this reproducible using the nightly builds or recent own compilation?

If I can find a `git` version of the package in AUR, I can see if the behavior persists when compiling it myself. Your nightlies don't seem to include Linux AppImage releases. (Also yes I've tried both the AppImage release (latest) and the package from my repos, and the behavior persists in both.)
I'll edit if I can get it compiled and see if it continues happening.
